dev-docs/source/Testing/MuonInterface/MuonTesting.rst

```diff
@@ -164,6 +164,73 @@ Transverse Field Asymmetry Test
 	- In the bottom table, check **TF Asymmetry Mode** to be ``true``
 	- Click **Fit**
 
+------------------------------------
+
+.. _single_pulse_test:
+
+Single Pulse Test
+-----------------
+
+**Time required 5 minutes**
+
+- Open **Muon Analysis** (*Interfaces* > *Muon* > *Muon Analysis*)
+- Change *Instrument* to **ARGUS**, found in the *Home* tab
+- Load run ``71799``
+- Go to the **Grouping** tab
```

Comment:
The whole line here is in bold, rather than just the text "Grouping". There are similar problems in the rest of this section and also in the Double Pulse Test section. I think it's something to do with the next line being indented (I've seen weird behaviour with that before).

![image](https://user-images.githubusercontent.com/12895056/118272563-b34cbe80-b4ba-11eb-8c0f-68e4ef403cca.png)

Reply:
Following offline discussions with Anthony, this is not ideal but is consistent with the rest of the documentation so it's not worth holding up the PR for